Documents/UseCasesForWebsite.txt

```diff
@@ -1,4 +1,26 @@
--  user wants to see the info page 
--  user wants to apply as artist 
-- user wants to donate 
--  user wants to have a prostictic worked on
\ No newline at end of file
+Actors: Visitor, Artist, Admin
+
+Visitor - View info page, View Gallery, Donate, RequestProstetic, Liking a Picture.
+A visitor would use this website for geting information or donate as amputee a prostetic or donate as a company.Also the visitors can visit the gallery and like one of the displayed art there.
+A visitor 
+- can also request that his/her prostetic is send to artist to make some art of it.
```

Comment:
**Request prosthetic**- what info needs to be filled, how to get to the request page?
Innacuarcy: prosthetic can either be requested with or without art. The requester might add sketches of art (or not) in case art is wanted.

Reply:
oke, but if a  prosthetic is requested without art. Why shoud the person then apply for this project. I thought and understand that this project is all about socks and  prosthetics with arts.

Reply:
It's all about helping people getting an implant. Regardless of decorations.

Painting is optional. 
Painting on UNUSED IMPLANTS and selling those is how they raise money.

You can still paint on actual implants, but not a must.

Reply:
It's all about helping people getting an implant. Regardless of decorations.

Painting is optional. 
Painting on UNUSED IMPLANTS and selling those is how they raise money.

You can still paint on actual implants, but not a must.